pkg/workceptor/kubernetes.go

```diff
@@ -256,17 +257,18 @@ func (kw *kubeUnit) kubeLoggingWithReconnect(streamWait *sync.WaitGroup, stdout
 				}
 
 				return
-			} else if _, ok := err.(http2.GoAwayError); ok {
+			} else if err != nil || goaway {
```

Comment:
wouldn't `err != nil` cover `goaway` also? do both these condition need to be here?

Reply:
Good point, I was playing with printing a specific `goaway` error, but never fully cleaned up once I was done. Ill change this now

Reply:
removed